Automattic/jetpack
# WordAds: Add translations to page parameters

#### Changes proposed in this Pull Request:
Adds translated text that will be used by the ad server javascript for inline + sticky slots. This mirrors what is currently available to dynamic ad slots.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Visit a site with ads enabled
* In JS console view that value of the ad server parameters `__ATA_JS`
* Verify the `ad` property exists and has translated `text` properties
